odata-mock/src/server/server.ts

```diff
@@ -1,19 +1,18 @@
 #!/usr/bin/env node
 'use strict'
 
-const odataApp = require("./app")
+import { init } from "./app"
 const app = require('express')()
-const LOGGER = require("./logger").logger
-
-var configPath
+import { logger as LOGGER } from "./logger"
 
 var runAsync = async () => {
+    var configPath: string = ""
     if (process.argv.length > 2) {
-        configPath = process.argv[2]
+
+        var configPath = process.argv[2]
```

Comment:
wouldn't the second configPath be local to the if only like this?

Reply:
yep ,fixed